examples/js/ShaderSkin.js

```diff
@@ -167,7 +167,7 @@ THREE.ShaderSkin = {
 				"diffuseColor = diffuseColor * colDiffuse;",
 
 				"vec3 normal = normalize( vNormal );",
-				"vec3 viewPosition = normalize( vViewPosition );",
+				"vec3 viewerDirection = normalize( vViewPosition );",
```

Comment:
This one would have been `viewDir`. I was torn between consistency and clarity and opted for the latter, figuring a reader might not necessarily be familiar with the in-core shading code. I would normally associate the "view direction" with a vector opposite to the "viewer direction". I have no strong opinion on it, just noting that it was done on purpose.

While talking about consistency: Does the inconsistent specular computation for hemispherical light sources in this file actually make sense?